README.md

```diff
@@ -0,0 +1,34 @@
+# Minetest Game
+
+The default game bundled in the Minetest engine.  
+For information about the game, click [here](https://forum.minetest.net/viewtopic.php?f=15&t=9724#p148074).
+
+## Installation
+
+- Unzip the archive, rename the folder to minetest_game and
+place it in ..minetest/games/
```

Comment:
`path_to_minetest/games`. `..` with a following separator indicates the parent directory, hence confusing.

Reply:
Please verify if it is correct.

Reply:
@SmallJoker i don't think it is as you requested,
```
- Unzip the archive, rename the folder to minetest_game and
place it in .. minetest/games/ ..
```
the following '..' is possibly confusing, as it should be placed in 'games' not '/..'?

Reply:
Like this?
```
 Unzip the archive, rename the folder to minetest_game and
place it in .. minetest/games ..
```

Reply:
> @SmallJoker
> 
> ```
> - Unzip the archive, rename the folder to minetest_game and
> place it in .. minetest/games/ ..
> ```
> 
> perhaps the following '..' is confusing, as it should be placed in 'games' not '/..'?

@paramat, should it be like this?

```
- Unzip the archive, rename the folder to minetest_game and 
place it in .. minetest/games ..
```
In my opinion, it looks better.

Reply:
'..' not after '/' doesn't make sense, as it indicates files or folders.
If you change this to
`place it in .. minetest/games/`
i'll +1 and merge it.

Reply:
@paramat, now done. 🙂